packages/api/scripts/cmds/db-types.js

```diff
@@ -14,6 +14,7 @@ export async function dbTypesCmd() {
 
   try {
     await dbSqlCmd({ cargo: true, testing: true })
+    await delay(2000)
```

Comment:
Is there a reason for this?

Reply:
Yes I was getting a socket hangup error. I think because the new postgrest version restarts itself when changes are made. It just needs a second or two to come back.